src/common/feedGenerator.ts

```diff
@@ -133,6 +133,15 @@ const getRawFiltersData = async (
         .andWhere('"userId" = $2')
         .andWhere(`status = '${ContentPreferenceStatus.Blocked}'`),
     ),
+    rawFilterSelect(con, 'words', (qb) =>
+      qb
+        .select(['"referenceId"'])
+        .from(ContentPreference, 'w')
+        .where('"feedId" = $1')
+        .andWhere(`type = '${ContentPreferenceType.Word}'`)
+        .andWhere(`status = '${ContentPreferenceStatus.Blocked}'`)
```

Comment:
For now we only care about blocked, eventually if we want reverse side we can move this to return status :)